src/main/scala/org/spurint/maven/plugins/mima/MiMaMojo.scala

```diff
@@ -79,12 +84,15 @@ class MiMaMojo extends AbstractMojo {
   @Parameter(property = "mima.skip", defaultValue = "false")
   private var skip: Boolean = false
 
-  @Parameter(property = "readTimeout", defaultValue = "4000")
-  private var readTimeout: Int = 4000
+  @Parameter(property = "readTimeout", defaultValue = "60000")
+  private var readTimeout: Int = 60000
```

Comment:
60 seconds is definitely too long; an unresponsive server will cause a long hang before timeout.  4 seconds may be too short, but please find a more reasonable value here, or revert to 4 seconds.